packages/riverpod_lint/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased minor
+
+- Added `async_value_nullable_patttern` lint, to warn against using `AsyncValue():final value?)` in pattern match when `value` is possibly nullable.
```

Comment:
likely a mistype on `AsyncValue():final value?)` and you meant `AsyncValue(:final value?)`

Reply:
Indeed thanks!

Reply:
Ty!